lib/db/api.js

```diff
@@ -349,4 +349,24 @@ dbapi.loadAccessToken = function(id) {
   return db.run(r.table('accessTokens').get(id))
 }
 
+dbapi.markStartOfDeviceUsage = function(leaseId, group, deviceMetadata, startsFrom, endsAt) {
+  return db.run(r.table('users').get(group.email))
+    .then(function(user) {
+        return db.run(r.table('deviceUsageStats').insert({
+          leaseId
```

Comment:
Unfortunately our codebase hasn't been set up for ES6 features yet and Travis is complaining about it. Would you mind changing all of these to the old-style `leaseId: leaseId` for now?